stl/inc/yvals_core.h

```diff
@@ -886,6 +889,54 @@
 
 // next warning number: STL4028
 
+// P0619R4 Removing C++17-Deprecated Features
+#ifndef _HAS_FEATURES_REMOVED_IN_CXX20
+#define _HAS_FEATURES_REMOVED_IN_CXX20 (!_HAS_CXX20)
+#endif // _HAS_FEATURES_REMOVED_IN_CXX20
+
+#ifndef _HAS_DEPRECATED_IS_LITERAL_TYPE
+#define _HAS_DEPRECATED_IS_LITERAL_TYPE (!_HAS_CXX20 || _HAS_FEATURES_REMOVED_IN_CXX20)

```

Comment:
Hmm, for each of the fine-grained macros, is `!_HAS_CXX20 ||` necessary? This seems overly complicated. I'd recommend simply defaulting to `(_HAS_FEATURES_REMOVED_IN_CXX20)`. It would allow someone in C++14/17 mode to remove all of these features with a single macro, instead of defining each fine-grained macro - that isn't something we're interested in encouraging, but it also doesn't seem worth extra complexity to make slightly harder.

Reply:
You are absolutely correct. Thanks.